cryoet_data_portal_neuroglancer/state_generator.py

```diff
@@ -221,11 +240,19 @@ def combine_json_layers(
     scale: tuple[float, float, float] | list[float] | float,
     units: str = "m",
     projection_quaternion: list[float] | None = None,
+    set_slices_visible_in_3d: bool | None = None,
+    show_axis_lines: bool = True,
 ) -> dict[str, Any]:
+    """Note, if set_slices_visible_in_3d is not provided, it will be set to False if there are any image layers in the list with volume rendering."""
     image_layers = [layer for layer in layers if layer["type"] == "image"]
+    if set_slices_visible_in_3d is None:
+        set_slices_visible_in_3d = not any(layer["volumeRendering"] == "on" for layer in image_layers)
+
     scale = get_scale(scale)
-    if not projection_quaternion:
+    if projection_quaternion is not None:
```

Comment:
I don't get so much this test, if `projection_quaternion` is not `None`, why is it set to another value in the body of the `if` that doesn't uses its value?

Reply:
Right I see what happened, thanks again for spotting this. The code is intended to set the projection quaternion to a default if no parameter is explicitly set. The previous check of `not projection_quaternion` was replaced because if you pass a numpy array (I know our type hint says tuple of floats, but the conversion to quaternion does work with array like objects) then `bool(numpy_array)` will fail. The explicit check for None solves this.

So this was meant to be `if proj_quat` is None: then set a default value. The bit afterwards with the `[]` was a debugging thing I forgot to remove, apologies on that.

See https://github.com/chanzuckerberg/cryoet-data-portal-neuroglancer/pull/25/commits/2692a6dde4fe3e7d410ceca288bbf0cb23e27733 which should resolve